KeyboardShortcutsExample/ContentView.swift

```diff
@@ -4,9 +4,102 @@ import KeyboardShortcuts
 extension KeyboardShortcuts.Name {
 	static let testShortcut1 = Self("testShortcut1")
 	static let testShortcut2 = Self("testShortcut2")
+	static let testShortcut3 = Self("testShortcut3")
+	static let testShortcut4 = Self("testShortcut4")
 }
 
-struct ContentView: View {
+struct Shortcut {
+	var name: KeyboardShortcuts.Name
+	let label: String
+}
+
+struct DynamicShortcutRecorder: View {
+	@Binding var name: KeyboardShortcuts.Name
+	@Binding var isPressed: Bool
+	var body: some View {
+		HStack {
+			KeyboardShortcuts.Recorder(for: name)
+				.padding(.trailing, 10)
+			Text("Pressed? \(isPressed ? "👍" : "👎")")
+				.frame(width: 100, alignment: .leading)
+		}
+	}
+}
+
+final class DynamicShortcutViewModel: ObservableObject {
```

Comment:
I think using a ViewModel for this is premature. This logic could be massively simplified and moved to the `DynamicShortcut` view.